BetterUI/Main.cs

```diff
@@ -98,146 +100,152 @@ public Main()
+            sectionName = "4 - Hover Text";
+
+            timeLeftStyleFermenter = Config.Bind(sectionName, nameof(timeLeftStyleFermenter), 2, "Select duration display. 0 = Default, 1 = % Done, 2 = min:sec left.");
 
-            timeLeftStylePlant = Config.Bind("4 - Hover Text", nameof(timeLeftStylePlant), 2, "Select duration display. 0 = Default, 1 = % Done, 2 = min:sec left");
+            timeLeftStylePlant = Config.Bind(sectionName, nameof(timeLeftStylePlant), 2, "Select duration display. 0 = Default, 1 = % Done, 2 = min:sec left.");
 
-            timeLeftStyleCookingStation = Config.Bind("4 - Hover Text", nameof(timeLeftStyleCookingStation), 2, "Select duration display. 0 = Default, 1= % Done, 2 = min:sec left");
+            timeLeftStyleCookingStation = Config.Bind(sectionName, nameof(timeLeftStyleCookingStation), 2, "Select duration display. 0 = Default, 1= % Done, 2 = min:sec left.");
 
-            chestHasRoomStyle = Config.Bind("4 - Hover Text", nameof(chestHasRoomStyle), 2, "Select how chest emptyness is displayed. 0 = Default | 1 = % | 2 = items / max_room. | 3 = free slots ");
+            chestHasRoomStyle = Config.Bind(sectionName, nameof(chestHasRoomStyle), 2, "Select how chest emptyness is displayed. 0 = Default | 1 = % | 2 = items / max_room. | 3 = free slots.");
 
 
             // Character XP
-            showCharacterXP = Config.Bind("5 - Character XP", nameof(showCharacterXP), true, "Enable Character XP. This combines all skill levels to show overall character progress.");
+            sectionName = "5 - Character XP";
 
-            showCharacterXpBar = Config.Bind("5 - Character XP", nameof(showCharacterXpBar), true, "Show Character XP Bar on the bottom of the screen. Character XP must be enabled.");
+            showCharacterXP = Config.Bind(sectionName, nameof(showCharacterXP), true, "Enable Character XP. This combines all skill levels to show overall character progress.");
 
-            showCharacterXpBar.Value = showCharacterXP.Value && showCharacterXpBar.Value;
+            showXPNotifications = Config.Bind(sectionName, nameof(showXPNotifications), true, "Show whenever you gain xp from actions.");
 
-            showXPNotifications = Config.Bind("5 - Character XP", nameof(showXPNotifications), true, "Show whenever you gain xp from actions.");
+            notificationTextSizeXP = Config.Bind(sectionName, nameof(notificationTextSizeXP), 14, "XP notification font size.");
 
-            notificationTextSizeXP = Config.Bind("5 - Character XP", nameof(notificationTextSizeXP), 14, "XP notification font size.");
+            extendedXPNotification = Config.Bind(sectionName, nameof(extendedXPNotification), true, "Extend notification with: (xp gained) [current/overall xp].");
 
-            extendedXPNotification = Config.Bind("5 - Character XP", nameof(extendedXPNotification), true, "Extend notification with: (xp gained) [current/overall xp]");
+            // Character XP RESTART
```

Comment:
Wouldn't it be better to keep this in the section it was in and add the "(Requires Logout)" in the setting description?

Reply:
see below for my reasoning